test/e2e/admin_portal.go

```diff
@@ -0,0 +1,215 @@
+		const CLUSTER_INFO_HEADINGS = 17
+		err := wd.Wait(ElementIsLocated(ByCSSSelector, "div[data-automation-key='name']"))
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		cluster, err := wd.FindElement(ByCSSSelector, "div[data-automation-key='name']")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		err = cluster.Click()
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		err = wd.Wait(ElementIsLocated(ByCSSSelector, "div.css-113"))
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		err = wd.WaitWithTimeout(ElementIsLocated(ByCSSSelector, "div.css-244"), 2*time.Minute)
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		err = wd.WaitWithTimeout(ElementIsLocated(ByCSSSelector, "div.css-244 > div > div > span"), 2*time.Minute)
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		panelSpans, err := wd.FindElements(ByCSSSelector, "div.css-244 > div > div > span")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		Expect(len(panelSpans)).To(Equal(CLUSTER_INFO_HEADINGS * 3))
+
+		panelFields := panelSpans[0 : CLUSTER_INFO_HEADINGS-1]
+		panelColons := panelSpans[CLUSTER_INFO_HEADINGS : CLUSTER_INFO_HEADINGS*2-1]
+		panelValues := panelSpans[CLUSTER_INFO_HEADINGS*2 : len(panelSpans)-1]
+
+		for _, panelField := range panelFields {
+			panelText, err := panelField.Text()
+			if err != nil {
+				TakeScreenshot(wd, err)
+			}
+
+			Expect(panelText).To(Not(Equal("")))
+		}
+
+		for _, panelField := range panelColons {
+			panelText, err := panelField.Text()
+			if err != nil {
+				TakeScreenshot(wd, err)
+			}
+
+			Expect(panelText).To(Equal(":"))
+		}
+
+		for _, panelField := range panelValues {
+			panelText, err := panelField.Text()
+			if err != nil {
+				TakeScreenshot(wd, err)
+			}
+
+			Expect(panelText).To(Not(Equal("")))
+		}
+	})
+
+	It("Should be able to copy cluster resource id", func() {
+		wd.Wait(ElementIsLocated(ByCSSSelector, "button[aria-label='Copy Resource ID']"))
+
+		button, err := wd.FindElement(ByCSSSelector, "button[aria-label='Copy Resource ID']")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		button.Click()
+
+		filter, err := wd.FindElement(ByCSSSelector, "input[placeholder='Filter on resource ID']")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		// Paste clipboard
+		filter.Click()
+		filter.SendKeys(ControlKey + "v")
+
+		resourceId, err := filter.GetAttribute("value")
+
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		Expect(resourceId).To(ContainSubstring("/providers/Microsoft.RedHatOpenShift/openShiftClusters/" + os.Getenv("CLUSTER")))
+	})
+
+	It("Should be able to open ssh panel and get ssh details", func() {
+		wd.Wait(ElementIsLocated(ByCSSSelector, "button[aria-label='SSH']"))
+
+		button, err := wd.FindElement(ByCSSSelector, "button[aria-label='SSH']")
+		if err != nil {
+			TakeScreenshot(wd, err)
+		}
+
+		button.Click()
+
+		wd.Wait(ElementIsLocated(ByID, "ModalFocusTrapZone25"))
+
+		wd.Wait(ElementIsLocated(ByID, "Dropdown55"))
```

Comment:
For the dropdown, we can pass `id` in to the props directly to set it:

```
<Dropdown
id="foo" 
```

Since there's only one SSH modal ,I don't think we need to randomise that.